pdr_backend/cli/cli_arguments.py

```diff
@@ -47,11 +47,12 @@
   pdr multisim PPSS_FILE
   pdr arima_plots PPSS_FILE [--debug_mode False]
   pdr deployer (for >1 predictoor bots)
+  pdr ohlcv PPSS_FILE NETWORK
   pdr lake raw|etl update PPSS_FILE NETWORK
   pdr lake raw|etl drop PPSS_FILE NETWORK ST
   pdr lake describe --HTML PPSS_FILE NETWORK
   pdr lake validate PPSS_FILE NETWORK
-  pdr analytics PPSS_FILE NETWORK
+  pdr predictoor_dashboard PPSS_FILE NETWORK
```

Comment:
the command for starting the predictoor dashboard is already mentioned inside the 'Main tools' section as `pdr dashboard` so let's just remove this line

Reply:
I deleted this command because it exists in "Main tools". This makes this PR 1:1 with what's inside of main branch

![image](https://github.com/user-attachments/assets/07b2de85-fbea-4673-afe7-66dfe8fd559d)